docn/cime_config/buildnml

```diff
@@ -11,15 +11,21 @@
 
 import os, sys, re
 
-_CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..","..")
-sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))
+_CDEPS_CONFIG = os.path.join(os.path.dirname(os.path.abspath(__file__)), os.pardir,os.pardir,"cime_config")
```

Comment:
See comments in datm - can we consolidate more of the repeated code in these files?